src/controller/jobRoleController.ts

```diff
@@ -10,7 +12,44 @@ export namespace JobRoles {
         } catch (e) {
             res.locals.errorMessage = e;
         }
-
         res.render("job-roles");
     }
+
+    export async function getEdit(req:Request, res:Response) : Promise<void> {
+        const { id } = req.params;
+        const jobRolesService: JobRole[] = await jobRoleService.getJobRoles();
+        const bands: JobBand[] = await bandService.getBands();
+        const capabilities: JobCapability[] = await capabilityService.getCapabilities();
+
+        const jobRoles = jobRolesService.find(i => i.jobRoleID === parseInt(id));
+        
+        res.render(`job-roles/edit`,{
+            jobRoles,
+            bands,
+            capabilities
+        })
+    }
+
+    export async function postEdit(req: Request, res: Response): Promise<void> {
```

Comment:
We need some validation of the user inputs. You are submitting unsanitised data to your API